packages/next/build/webpack/loaders/next-babel-loader.js

```diff
@@ -58,6 +58,7 @@ module.exports = babelLoader.custom(babel => {
         pagesDir: opts.pagesDir,
         hasModern: opts.hasModern,
         babelPresetPlugins: opts.babelPresetPlugins,
+        development: opts.development,
```

Comment:
Should we also increment the `cacheKey`?

Reply:
Nope, this will auto cause invalidation. The cache key is meant for babel dep bumps!
https://github.com/zeit/next.js/pull/9610/files#diff-ed38aaf06b09b233f67d0b7e6616a7c6R74